nixos/modules/services/web-apps/documenso.md

```diff
@@ -0,0 +1,41 @@
+# Documenso {#module-services-documenso}
+
+Documenso is an open-source digital signature platform, offering a
+cost-effective and customizable alternative to proprietary solutions like
+DocuSign.
+
+## Quickstart {#module-services-documenso-quickstart}
+
+To run documenso in a non-production environment use this configuration.
+
+```
+{
+  services.documenso = {
+    enable = true;
+  };
+}
+```
+
+If you want to create some initial database content including an admin user run
+the following command:
+
+```ShellSession
+systemctl start documenso-seed-database
+```
```

Comment:
Why is that not done automatically?

Reply:
The seed seem to set default password for the admin which is not great security wise, so this is not made the default to prevent vulnerability by mistake